include/seqan3/range/view/trim.hpp

```diff
@@ -209,17 +209,17 @@ namespace seqan3::view
  *
  * Or operating on a range of seqan3::dna5q:
  * ```cpp
- * std::vector<dna5q> vec{{dna5::A, illumina18{40}}, {dna5::G, illumina18{40}}, {dna5::G, illumina18{30}},
- *                        {dna5::A, illumina18{20}}, {dna5::T, illumina18{10}}};
- * std::vector<dna5q> cmp{{dna5::A, illumina18{40}}, {dna5::G, illumina18{40}}, {dna5::G, illumina18{30}},
- *                        {dna5::A, illumina18{20}}};
+ * std::vector<dna5q> vec{{dna5::A, qualified{40}}, {dna5::G, qualified{40}}, {dna5::G, qualified{30}},
+ *                        {dna5::A, qualified{20}}, {dna5::T, qualified{10}}};
+ * std::vector<dna5q> cmp{{dna5::A, qualified{40}}, {dna5::G, qualified{40}}, {dna5::G, qualified{30}},
+ *                        {dna5::A, qualified{20}}};
```

Comment:
I think you did a mistake in the search and replace here, it was supposed to be phred42, right?

Reply:
Yes, with the next commit it will be replaced with phred42.